python/python/glide/async_commands/core.py

```diff
@@ -694,6 +694,29 @@ async def lpush(self, key: str, elements: List[str]) -> int:
             int, await self._execute_command(RequestType.LPush, [key] + elements)
         )
 
+    async def lpushx(self, key: str, elements: List[str]) -> int:
+        """
+        Inserts specified values at the head of the `list`, only if `key` already exists and holds a list.
+
+        See https://redis.io/commands/lpushx/ for more details.
+
+        Args:
+            key (str): The key of the list.
+            elements (List[str]): The elements to insert at the head of the list stored at `key`.
+
+        Returns:
+            int: The length of the list after the push operations.
+
+        Examples:
+            >>> await client.lpushx("my_list", ["value1", "value2"])
+                3 // Indicates new length of the list
```

Comment:
```suggestion
                3 # Indicates that 2 elements we're added to the list "my_list", and the new length of the list is 3.      
```

Reply:
Oh thanks, I used to use java comments with //